### DISCUSSION TITLE
OREGON SCIENTIFIC THGN301

### DISCUSSION THREAD

**Discussion by vgeannin** [merbanan/rtl_433#2623] [Q&A]
Hi All,

Do you know if this sensor work with rtl_433 : [THGN301](https://www.oregonscientificstore.com/p-396-oregon-scientific-thgn301-extra-temperature-and-humidity-sensor-for-wmr300.aspx)

It's a 868Mhz sensor in europe

Thanks

**Comment by zuckschwerdt** [comment]
These are supported, other can be added with a little work.
https://github.com/merbanan/rtl_433/blob/70d84d01e1be87b459f7a10825966f3262b7dd34/src/devices/oregon_scientific.c#L18-L48
